src/axelrod_dojo/archetypes/cycler.py

```diff
@@ -13,15 +13,15 @@ class CyclerParams(Params):
     genetic lifecycle.
 
     **********
-    TODOs:
+    Expansions:
```

Comment:
If you don't plan to do this in the current CL please remove the comment. You can open an issue in the issue tracker instead.